adapter/application.go

```diff
@@ -0,0 +1,35 @@
+package adapter
```

Comment:
Are there new unit tests for this?

Reply:
There are none as of now. As the implementation is fleshed out, we will work on the testing strategy that may include unit tests.